service/src/main/java/bio/terra/cbas/controllers/RunSetsApiController.java

```diff
@@ -403,20 +401,6 @@ public static List<String> validateRequestInputsAndOutputs(
     return errorList;
   }
 
-  private String getErrorMessage(ApiException exception) {
-    Gson gson = new Gson();
-    try {
-      ErrorResponse error = gson.fromJson(exception.getResponseBody(), ErrorResponse.class);
-      if (error != null) {
-        return error.getMessage();
-      } else {
-        return exception.getMessage();
-      }
-    } catch (Exception e) {
-      return exception.getMessage();
-    }
-  }
-
```

Comment:
the new version of this moved to `WdsClientUtils` so it can be reused from multiple places